README.md

```diff
@@ -23,350 +22,80 @@
 <img src="assets/github-hr.png" alt="---" />
 <br/>
 
-**k6** is a modern load testing tool, building on [our](https://k6.io/about) years of experience in the load and performance testing industry. It provides a clean, approachable scripting API, [local](https://k6.io/docs/getting-started/running-k6) and [cloud execution](https://k6.io/docs/cloud), and flexible configuration.
+**k6** is a modern load-testing tool, built on [our years of experience](https://k6.io/about) in the performance and testing industries.
+k6 is built to be powerful, extensible, and full-featured. The key design goals are **high performance and the best developer experience**.
```

Comment:
```suggestion
k6 is built to be powerful, extensible, and full-featured. The key design goals are **the best developer experience** and high performance.
```
Based on the last company-wide discussion and the different perceptions of performance we got I guess we should at least reverse the order or reduce the emphasis on high performance. If we want to have it (I can understand that) then maybe should we explain it a bit more?

Reply:
This is a good point. I've just removed the "high performance."

This brings up a huge topic: What are the key design goals of k6? What gets sacrificed when a tradeoff has been made between performance and ease of use? Why implement libraries instead of having testers write their own helper functions.

These questions probably require participation from a large amount of stakeholders.
But, this discussion is too important to have in a side conversation about a Readme fix. I'm going to make a side issue.